app/app_main.c

```diff
@@ -2095,12 +2091,23 @@ static ACVP_RESULT app_hmac_handler(ACVP_TEST_CASE *test_case)
     return ACVP_SUCCESS;
 }
 
+void printBytes(unsigned char *buf, size_t len) {
```

Comment:
Left over debug ?

Reply:
Oh whoops, thought I left that out :) I'll fix that!

Reply:
Fixed!